src/sentry/static/sentry/app/components/onboardingWizard/getOnboardingTasks.tsx

```diff
@@ -85,8 +87,8 @@ export default function getOnboardingTasks(organization: Organization): Onboardi
       ),
       skippable: true,
       prereq: [1, 2],
-      featureLocation: 'project',
-      location: 'settings/release-tracking/',
+      actionType: 'app',
+      location: `/settings/${organization.slug}/projects/:projectId/release-tracking/`,
```

Comment:
Same as above. What should happen when the first step (creating a project) hasn't been completed yet?

Reply:
Ahh, we should rethink these to be disabled in that case. I'm okay just having the selector be empty in that case.